### ISSUE TITLE
Missing Documentation

### ISSUE THREAD

**Issue by FLAMESpl** [WattleScript/wattlescript#69]
There isn't much in the readme and project does not have a real documentation.

I am wondering how to setup sandboxing for my scripts:

- how do I avoid stack overflows?
- how do I limit memory usage?

And also how to interop with .NET, whether it is possible in LUA mode too or only in WattleScript?

**Comment by MirKml** [comment]
agree, there is nothing about Wattle language - without LUA mode. I'm interested in, because there isn't many script solutions for .NET. Moonsharp seems dead.

**Comment by FLAMESpl** [comment]
@MirKml Interesting alternative is scripting in c# (or generally in any language) then compile it to WASM in runtime (there is wasm-time port for dotnet). wasm-time allows you to run script in completely isolated environment and can interop with the host.
